.goreleaser.yaml

```diff
@@ -43,6 +43,31 @@ archives:
     format_overrides:
       - goos: windows
         format: zip
+nfpms:
+  - file_name_template: >-
+      {{ .PackageName }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
+    contents:
+      - src: internal/release/dynatrace-otel-collector.service
+        dst: /lib/systemd/system/dynatrace-otel-collector.service
+      - src: internal/release/dynatrace-otel-collector.conf
+        dst: /etc/dynatrace-otel-collector/dynatrace-otel-collector.conf
+    umask: 0o002
+    scripts:
+      preinstall: internal/release/preinstall.sh
+      postinstall: internal/release/postinstall.sh
+      preremove: internal/release/preremove.sh
```

Comment:
Probably doesn't matter but I think the release dir could even live at root rather than under internal.

Reply:
ah i don't know. i thought about it but it seems too prominent to me